src/views/ops/Template/components/Variable.vue

```diff
@@ -68,6 +68,9 @@ export default {
     variables: {
       handler(value) {
         if (value.length > 0 || this.initial) {
+          value.map((item) => {
+            item.default_value = item.text_default_value || item.select_default_value
+          })
           this.$emit('input', value)
         }
         if (value) {
```

Comment:
There is an added 'item' parameter inside the map function, which isn't specified anywhere else in your code snippet. Could you please confirm that this variable `item` exists somewhere within your application context? If not, perhaps it's used incorrectly or it has some other role to play elsewhere in your system; otherwise this change may have unintended side effects.

Additionally, ensure that all properties of your objects (specifically those with `.default_value` attributes), including initial values, are defined correctly. This includes ensuring valid data types (`string`, `number`, etc.) and that there aren't typos making them incompatible with their original purposes. The changes were made according to my knowledge cut-off. As of September 2021, these rules still apply generally.